app/main.py

```diff
@@ -1,3 +1,13 @@
 def cache(func):
-    # Write your code here
-    pass
+    results = {}
+
+    def inner(*args):
+        nonlocal results
```

Comment:
this statement is redundant, as dict is mutable type

Reply:
I have a very old laptop. It didn't work without this line yesterday, but it works today. It's miracle or i don't know what is it:)